python/change-notes/2021-07-28-port-RoDoS-queries.md

```diff
@@ -0,0 +1,3 @@
+lgtm,codescanning
+* Ported _Inefficient regular expression_ (`py/redos`) query from javascript.
+* Ported _Polynomial regular expression used on uncontrolled data_ [`py/polynomial-redos`] query from javascript.
```

Comment:
Not sure customers will care that we ported this, but I think it's fair to highlight that this is already available in JS. (but this is mostly just a nitpick)

```suggestion
* Added _Inefficient regular expression_ (`py/redos`) query, which is already available in JavaScript.
* Added _Polynomial regular expression used on uncontrolled data_ (`py/polynomial-redos`), which is already available in JavaScript.
```